addOns/alertFilters/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## [12] - 2021-09-16
 ### Added
 - Support for the Automation Framework
+- Stats for alerts changed
```

Comment:
Needs to be moved to the unreleased section.

Reply:
Doh! Fixed :)